### PR TITLE
[WIP] RBAC Full Audit on Cloud Load Balancers

### LINKED ISSUE #1783: Full Audit: Permissions Matrix for Cloud Load Balancers
https://github.com/rackerlabs/rackspace-how-to/blob/master/content/cloud-load-balancers/permissions-matrix-for-cloud-load-balancers.md
- [x] SME check: Update any out of date or inaccurate technical content with the help of a SME
- [x] Editor check: Incorporate edits from Kelly Holcomb
- [ ] Quality Checklist: Perform a quality audit of the article using the quality checklist
- [ ] Update audit_date: Update the audit_date metadata in the article with the date of the full audit's completion

### PR DESCRIPTION
**Pull request by nimbinatus**
Checking against API.

Fixes #1783

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1aa5d8a05f/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6dc7d33f09/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b5abebf9f9/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-78648c7e18/how-to/)

**Comment by nimbinatus** [comment]
email sent to potential SME (Jorge)

**Comment by filler** [comment]
Hey all.  This came to me by way of Trent by way of Jorge.  Ive reached out to the CLB product team (dev, ops, qe) for product eyeballs on the proposed change.  Its being worked in #lbaas on Rack IRC.

**Comment by nimbinatus** [comment]
Perfect; thanks @filler !

File: content/cloud-load-balancers/permissions-matrix-for-cloud-load-balancers.md [hunk 1]
```diff
@@ -1,155 +1,187 @@
...
-An Internet Protocol (IP) configured on the load balancer for use by clients connecting to a service that is load balanced. Incoming connections are distributed to back-end nodes based on the configuration of the load balancer.
+The Cloud Load Balancers permissions matrix displays specific permissions for the following role-based access control (RBAC) roles:
+
+-  **Admin** provides full access to create, read, update, and delete.
+-  **Creator** provides access to create, read, and update.
+-  **Observer** provides read-only access.
+
+ The matrix displays the Cloud Load Balancers methods grouped by category, their corresponding RESTful API commands, and the RBAC roles that are supported.
+
+**Warning:** If SSL is enabled on a load balancer configured with nodes that are _not_ in the same datacenter, then decrypted traffic will be sent in clear text over the public Internet to the external nodes and will no longer be secure.
```
Comment (the2hill) [inline content/cloud-load-balancers/permissions-matrix-for-cloud-load-balancers.md]: I'm not sure this warning belongs here. This should have been something in the API ref and the General info sections. Not sure what it has to do with permissions.

**Comment by the2hill** [comment]
Looks good to me except for the warning I'm unsure about. That was there from before. I suppose it's not hurting anyone leaving it there, just doesn't make a whole lotta' sense.

File: content/cloud-load-balancers/permissions-matrix-for-cloud-load-balancers.md (hunk 1 above)
Comment (nimbinatus) [inline content/cloud-load-balancers/permissions-matrix-for-cloud-load-balancers.md]: @the2hill, it was just pulled directly from the API docs. I'll remove it.

**Comment by nimbinatus** [comment]
@the2hill, thanks for the review! I can certainly remove the warning; we probably pulled it in from API just to have it. We've removed a lot of the unnecessary stuff like that in other RBAC matrixes, so I'll remove it here.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-748df8b29c/how-to/)

**Comment by nimbinatus** [comment]
@kmsholcomb, sorry, here's the actual PR for that issue I tagged you on. Can you add it to your queue? Thanks!

**Comment by kmsholcomb** [comment]
Hi @nimbinatus. Edits attached. I suggested taking out some notes that don't seem really necessary in the RBAC article (feel free to leave them in if you feel otherwise), and I had a few questions about why certain roles are or aren't listed for certain operations. Other than that, just some nitpicky edits. Please let me know if you have any questions. Thanks.

[Permissions matrix for Cloud Load Balancers_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/749171/Permissions.matrix.for.Cloud.Load.Balancers_editsKH.pdf)
[Permissions matrix for Cloud Load Balancers_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/749172/Permissions.matrix.for.Cloud.Load.Balancers_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-95a77c7c34/how-to/)

**Pull request merged**